src/nvim/api/private/helpers.c

```diff
@@ -84,6 +85,103 @@ bool try_end(Error *err)
   return ERROR_SET(err);
 }
 
+bool attr2hlattr(int attr_code, bool use_rgb, HlAttrs *out)
```

Comment:
helpers.c is turning into misc1.c... Can't these functions live in some more appropriate module?